### PR TITLE
DiscoFlix Sandbox Role Documentation

### PR DESCRIPTION
**Pull request by nickheyer**
Providing documentation for DiscoFlix - a role currently in [PR in the Sandbox repo](https://github.com/saltyorg/Sandbox/pull/273)

For new Sandbox app/role documentation, please confirm you have completed the following tasks:

- [X] App documentation page created
- [X] markdownlint reports no errors on edited page(s) - use <https://dlaa.me/markdownlint/> if your IDE does not have this built in
- [X] Reference added in `docs/sandbox/index.md`
- [X] Reference added in `mkdocs.yml` nav menu

### CODE REVIEW

File: docs/sandbox/apps/discoflix.md
Comment (chazlarson) [inline docs/sandbox/apps/discoflix.md:1]: Personally I'm not fond of marketing language in these docs, because it sounds like choosing sides.

Comment (nickheyer) [reply docs/sandbox/apps/discoflix.md:1]: Understood, i'll tone it down as much as I can.

Comment (nickheyer) [reply docs/sandbox/apps/discoflix.md:1]: Revised, let me know if there is anything else I can do.

**Review by chazlarson — APPROVED** [review]

**Comment by maximuskowalski** [comment]
Role has now been merged so I'll merge these docs as well.

**Comment by nickheyer** [comment]
@maximuskowalski 

Did I forget to include something in my docs? They aren't "included in the nav configuration". Visibile when searched in the searchbar, but not in the sandbox apps tab.
 
https://github.com/saltyorg/docs/actions/runs/5839253726/attempts/1#summary-15837214588

**Comment by maximuskowalski** [comment]
Was this not what you intended? Or am I misunderstanding? 

image

**Pull request merged**